spec/system/consumer/split_checkout_spec.rb

```diff
@@ -68,12 +67,13 @@
       order.update(state: "payment")
       get checkout_step_path(:details)
       expect(response).to have_http_status(:redirect)
-      expect(page).to have_current_path("/checkout/guest")
+      expect(page).to have_content("Ok, ready to checkout?")
     end
 
     it "should redirect to the login page when clicking the login button" do
       click_on "Login"
-      expect(page).to have_current_path "/"
+      expect(page).to have_content("Login")
+      expect(page).to have_content("Login")
```

Comment:
Once is enough :wink:

Reply:
Hehe, thanks :-)
Need to change that and rubocop error... 

I was thinking of [addressing this comment](https://github.com/openfoodfoundation/openfoodnetwork/pull/8728#issuecomment-1015239221) before moving to code review, but maybe it is best to do it a seperate PR...

Reply:
Yeah it could be a separate PR. Also worth noting: the country/state selectors on the split checkout don't work the same way and don't make any API calls.

Reply:
Ok, this PR would be ready for review now.

It could be a good idea to test the country/state selectors on split checkout. I tried mocking other countries to have more options on the dropdown, but could not really make that work yet. 

So, to be addressed on separate PRs:
- test country/state selectors
- test customer creation/updating

And then of course, continuing with step 2 and 3.